### DISCUSSION TITLE
Anyone tried this on Ubuntu Subsystem on Windows (WSL2)?

### DISCUSSION THREAD

**Discussion by DatN132** [madMAx43v3r/chia-plotter#159] [Q&A] [answered]
As this works fine on Ubuntu, have anyone tried running it on Windows via WSL? 
If it works, then how much performance do you lose?

**Comment by delucca** [comment, marked as answer]
To run in Windows the easiest way is to use a Docker container, like explained in #102

**Reply by DatN132** [reply]
Hmm. Interesting. Thanks. It seems that docker does not incur that much performance penalty.

**Reply by delucca** [reply]
As far as I know, yes

Also, it seems reasonable to assume this, since most of web applications run inside Docker in a Kubernetes container.
